src/storage/heap_file.c

```diff
@@ -15832,6 +15832,34 @@ heap_rv_redo_insert (THREAD_ENTRY * thread_p, const LOG_RCV * rcv)
       return er_errid ();
     }
 
+  if (is_passive_transaction_server ())
+    {
+      if (recdes.type != REC_ASSIGN_ADDRESS)
+	{
+	  /* Classname cache (locator_Mht_classnames) status should be reverted when
```

Comment:
It would be good to me if the comment also explain why the status of classname entry should be changed.

Reply:
Stating that it reverts due to rollback seemed sufficient. Are you interested in a description of what happens if a revert is not performed?

Reply:
I think if you add a description why the cache status should be changed in front of the locator_change_classname_entry_status_if_exists definition, then you can cover all of my request about comment modification. I expect you to describe it based on the DROP statement and the your reply as below: 

> The classname cache can be accessed regardless of whether a lock is acquired. In a read transaction, access to the cache is made to obtain the class OID corresponding to the classname. If the cache status for that class is LC_CLASSNAME_EXIST, a check is performed on the actual heap to see if a record exists using the class OID

Reply:
OK, I will